Lib/ctypes/util.py

```diff
@@ -324,6 +322,20 @@ def test():
             print(cdll.LoadLibrary("libcrypto.dylib"))
             print(cdll.LoadLibrary("libSystem.dylib"))
             print(cdll.LoadLibrary("System.framework/System"))
+        # issue-26439 - fix broken test call for AIX
+        elif sys.platform.startswith("aix"):
+            from ctypes import CDLL
+            RTLD_MEMBER =  0x00040000
+            # print("crypto\t:: %s" % cdll.LoadLibrary(find_library("crypto")))
+            if (sys.maxsize < 2**32):
```

Comment:
Round brackets have re-appeared

Reply:
Do you mean this should be:
if {sys.maxsize < 2**32}:

Confused! by comment.

Reply:
It would be easier to read (IMO) without any kind of brackets. I removed them when I modified your patch (https://bugs.python.org/review/26439/diff2/18236:18714/Lib/ctypes/util.py), and they stayed removed in your updated aix-library.161004.patch. See also test_loading.py.